app/views/layouts/_sorting_bar.haml

```diff
@@ -0,0 +1,34 @@
+%nav.navbar.bg-light.sorting-nav
+  .mr-auto
+    .dropdown
+      %button.dropbtn
+        Category
+        %i.fa.fa-angle-down
+      / .dropdown-content
+      /   - @categories = @categories.nil? ? Category.all : @categories
+      /   - @categories.each do |category|
+      /     %a{:href => category_path(category)}= category.title
+    .dropdown
+      %button.dropbtn
+        Sorting
+        %i.fa.fa-angle-down
+      .dropdown-content
+        %a{ :href => '#' } Link 1
+    .dropdown
+      %button.dropbtn
+        Vegetarian
+        %i.fa.fa-angle-down
+      .dropdown-content
+        %a{ :href => '#' } Link 1
```

Comment:
where does this lead?

Reply:
That is an example dropdown in the sorting bar, since on the example design it had to have three buttons for sorting. It's just a placeholder to describe the functionality. It does, indeed, not lead anywhere.

Reply:
change to link_to